packages/middleware/src/apiClientFactory/applyContextToApi.ts

```diff
@@ -36,6 +38,17 @@ const applyContextToApi = <
         const extendQuery = createExtendQuery(context);
         const transformedArgs = await hooks.before({ callName, args });
         const apiClientContext = { ...context, extendQuery };
+        if (isCustomEndpointHandler(fn)) {
+          if (context.res.locals?.alokai?.metadata?.scope) {
+            context.res.locals.alokai.metadata.scope.extensionName =
+              fn._extensionName;
+          } else {
+            const logger = getLogger(context.res);
+            logger.warning(
```

Comment:
Should be a debug imo, ant the info can be more technical as this is a very meaningless information that will be printed for every request eventually.
```suggestion
            logger.debug(
```

Reply:
It means something is incorrect internally. It decreases the quality of metadata. A client should act so I think warning is perfectly fine, as app is still working but they need to act.